### PR TITLE
Add support for RISC-V 64

### PR DESCRIPTION
**Pull request by clansty**

### CODE REVIEW

File: debian/patches/10_riscv_support.patch [hunk 1]
```diff
@@ -0,0 +1,70 @@
+Index: xutils-dev/xorg-cf-files/Imake.cf
```
Comment (RevySR) [inline debian/patches/10_riscv_support.patch]: 著名From author  update
https://cgit.freedesktop.org/xorg/util/cf/patch/?id=027da8ee0f3628078e5fd364a4d67fa84a35b304

Comment (clansty) [reply debian/patches/10_riscv_support.patch]: 其实是 https://github.com/bmwiedemann/openSUSE/blob/4eded4be9125b3e8b4e2e0730d07b4f1e536b052/packages/x/xorg-cf-files/u_riscv.patch

**Review by RevySR — APPROVED** [review]

**Pull request merged**